dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/TrackedEntityInstanceFilterController.java

```diff
@@ -38,7 +44,34 @@
  */
 @Controller
 @RequestMapping( value = TrackedEntityInstanceFilterSchemaDescriptor.API_ENDPOINT )
+@ApiVersion( include = { DhisApiVersion.ALL, DhisApiVersion.DEFAULT } )
```

Comment:
Do we really need this annotation here?

Reply:
If the annotation is missing, api/{number}/rest/Of/the/url will not work. So I just made it to be standard. this means even if the api is accesed with or without the apiVersion, it continues to reach here.